DOCSP-49150: bulkWrite docs + api (#250)
mongodb/docs-php-library: source/whats-new.txt

Learn about new features, improvements, and fixes introduced in the
following versions of the {+php-library+}:

* :ref:`Version 2.1 <php-lib-version-2.1>`
* :ref:`Version 2.0 <php-lib-version-2.0>`
* :ref:`Version 1.21 <php-lib-version-1.21>`
* :ref:`Version 1.20 <php-lib-version-1.20>`
* :ref:`Version 1.19 <php-lib-version-1.19>`
* :ref:`Version 1.18 <php-lib-version-1.18>`
* :ref:`Version 1.17 <php-lib-version-1.17>`

.. _php-lib-version-2.1:

What's New in 2.1
-----------------

The {+library-short+} v2.1 release includes the following features,
improvements, and fixes:

> - Implements a *client* bulk write API that allows you to perform write

Reviewer: I suggest this simpler language.

```suggestion
- Add a *client* bulk write API that perform write
```
